src/Castle.Facilities.WcfIntegration.Tests/Client/WcfClientFixture.cs

```diff
@@ -267,6 +267,9 @@ public void CanResolveClientAssociatedWithChannelUsingSuppliedModel()
 		}
 
 		[Test]
+#if DEBUG
+		[Ignore("This test fails in Debug / relies on GC.Collect to clean weak reference")]
```

Comment:
As far as I know, in Debug mode the JIT will choose to keep the local variable alive until the end of the method, so that it can be inspected in the debugger. This is not new, it's likely this test was failing before.